### ISSUE TITLE
Validate-Job returns an error when the optional document-format attribute is not included in the request

### ISSUE THREAD

**Issue by michaelrsweet** [istopwg/ippsample#18]
Reported by Smith via email - when ippserver isn't configured with a default document-format value, the Validate-Job request sent by a CUPS client is rejected based on the lack of a document-format value, which is a bug.

**Comment by wifiprintguy** [comment]
The root cause seems to be that ippserver assumes "document-format-default" = 'application/octet-stream' if it isn't explicitly specified, but won't find it in "document-format-supported" unless the user has explicitly specified that. If "document-format-default" is explicitly specified to be one of the document formats in "document-format-supported", or if "document-format-supported" includes 'application/octet-stream', then the bad behavior isn't observed and the Validate-Job operation is successful.

In other words, this will fail:

ATTR mimeMediaType    document-format-supported    image/jpeg,image/pwg-raster

but adding this will cause it to start working:

ATTR mimeMediaType   document-format-default   image/pwg-raster

or alternately appending 'application/octet-stream' to the list of supported document formats like so:

ATTR mimeMediaType    document-format-supported    image/jpeg,image/pwg-raster,application/octet-stream

**Comment by michaelrsweet** [comment]
[master 952fc03] Don't require/test document-format for Validate-Job operation (Issue #18)
